app/index.jsx

```diff
@@ -33,7 +33,7 @@ const getLanguageStorage = function(){
 };
 
 const connectionManager = new ConnectionManager({
-    appId: 1,
+    appId: 12812,
     language: getLanguageStorage(),
     endpoint: 'wss://frontend.binaryws.com/websockets/v3',
```

Comment:
Change WS URL to `wss://ws.binaryws.com/websockets/v3`

Reply:
@ashkanx done.